scripts/kvt_genvk.py

```diff
@@ -26,15 +26,14 @@ def startTimer(timeit):
 
 def endTimer(timeit, msg):
     global startTime
-    endTime = time.clock()
     if timeit:
         endTime = time.process_time()
         write(msg, endTime - startTime, file=sys.stderr)
         startTime = None
 
 # Turn a list of strings into a regexp string matching exactly those strings
 def makeREstring(list, default = None):
-    if len(list) > 0 or default == None:
+    if len(list) > 0 or default is None:
```

Comment:
mild suggestion: this is slightly more Pythonic:
```
if list or default is None:
```

Reply:
This file is forked from the upstream Vulkan-Docs repo https://github.com/KhronosGroup/Vulkan-Docs/blob/7b81afeb9f76ed8b1ca6c2ed9c93d94eee08bfed/scripts/genvk.py#L53